internal/daemon/worker/testing.go

```diff
@@ -445,6 +445,11 @@ func NewTestMultihopWorkers(t testing.TB,
 		WorkerAuthDebuggingEnabled: enableAuthDebugging,
 		DownstreamWorkerAuthKms:    childDownstreamWrapper,
 	})
+	t.Cleanup(kmsWorker.Shutdown)
+
+	// Give time for it to be inserted into the database
+	time.Sleep(2 * time.Second)
+
```

Comment:
I think make-gen-delta is complaining about this extra line here

Reply:
that's what I get for porting the patch over in vim